### ISSUE TITLE
Deeptrio pileup height being incorrectly set and missing openvino

### ISSUE THREAD

**Issue by ASLeonard** [google/deepvariant#416]
Hi,
This is all done using the singularity containers of the 1.1.0 version, on a linux system. 

I encountered one error and one potential error when experimenting with DeepTrio. The first potential error is mismatched pileup heights, giving the following error when running the single command deeptrio under the PACBIO model
` call_variants.py:355] The height of the input image is not 100 (standard in DeepVariant) or 300 (standard in DeepTrio).`

There are references to the pileup height defaulting to 0 and thus taking the value in **dv_constants.py**, but it appears in the **run_deeptrio.py** script [here](https://github.com/google/deepvariant/blob/5bee15713085316ae706c7f61274cd438b0006dc/scripts/run_deeptrio.py#L315), that the pileup is set to 60 and 40 **unless** being run in WES mode, then it is 100 each. I then reran the make_examples with explicit pileupheights for child and parent as 100, and the warning went away.

The second issue is using openvino in deeptrio. When checking the **deeptrio_metric.md** doc, I didn't see any reference to openvino, so it may not be fully supported still, but it is a command line option. However, when using the same style of command that I got working in #404, I got the following error.

```
 File "/tmp/Bazel.runfiles_fmlnwqjy/runfiles/com_google_deepvariant/deepvariant/call_variants.py", line 409, in call_variants
    checkpoint_path, input_fn=tf_dataset, model=model)
  File "/tmp/Bazel.runfiles_fmlnwqjy/runfiles/com_google_deepvariant/deepvariant/openvino_estimator.py", line 89, in __init__
    freeze_graph(model, checkpoint_path, tensor_shape)
  File "/tmp/Bazel.runfiles_fmlnwqjy/runfiles/com_google_deepvariant/deepvariant/openvino_estimator.py", line 77, in freeze_graph
    graph_def = optimize_for_inference_lib.optimize_for_inference(
NameError: name 'optimize_for_inference_lib' is not defined
```

The error is a bit misleading, as that import is valid, but in a try/except block [here](https://github.com/google/deepvariant/blob/2dbebb4d97e15d0d5fcf303a4466314b1f313208/deepvariant/openvino_estimator.py#L37) which actually fails because openvino is not installed in the container. 

In the deepvariant image, there is `/opt/intel/openvino/`, while there is nothing like that in the deeptrio image.

Thanks,
Alex

**Comment by ASLeonard** [comment]
I was just double checking the openvino issue in case I missed something obvious, but there are explicitly `openvino_estimator` references present in the deepvariant BUILD file, but not in the deeptrio BUILD file.

**Comment by danielecook** [comment]
@ASLeonard 

The observed warning is not correct and is an issue we are working on fixing. Currently, DeepTrio will report that warning for all models except for WES. However, it does not impact DeepTrio inference.

We are also planning on adding OpenVINO support to the next release of DeepTrio.

**Comment by ASLeonard** [comment]
Great, so it seems there is no need to manually specify `--pileup_image_height_*=100`. Thanks for the details.

**Comment by ASLeonard** [comment]
Did openvino make it into deeptrio-1.2.0? From some quick tests, it doesn't look like it.

**Comment by pichuan** [comment]
Thanks @ASLeonard for checking in. It was originally on our roadmap to add OpenVINO to DeepTrio. However, when we were updating DeepVariant's metrics.md, we found that with OpenVINO runs roughly the same as without it. ( (We observed a pretty significant speedup in r1.2 using just intel-tensorflow without OpenVINO.)
As a result, I didn't add OpenVINO to DeepTrio this time.

@ASLeonard You can still find OpenVINO in our DeepVariant Docker image. If you still observe a noticeable speedup using OpenVINO using our DeepVariant setting, let me know.
I'd really love to hear what type of machine you're using. FYI for @dkurt because this is OpenVINO related.

**Comment by ASLeonard** [comment]
These mostly run on nodes with Intel Xeon Gold 6140, occasionally on Intel Xeon E5-2697v4, but these are much slower anyway. 

I had noticed the same speed improvement in v1.1.0 with openvino as your metrics, but I haven't tested the new version extensively with and without. However, when rerunning identical samples (both with openvino flags), I've noticed that v1.2.0 takes longer wall clock time compared to v1.1.0, but less CPU time. Maybe it is just node variation or other jobs bottlenecking IO, so hopefully will see a clearer result after more samples run.
